### DISCUSSION TITLE
How to send text selection using collaboration cursor?

### DISCUSSION THREAD

**Discussion by nc** [ueberdosis/tiptap#1847] [General Questions] [answered]
I've got the collaboration cursor working but it doesn't highlight the users text selection like the homepage demo does?

How does one send and render the current text selection for a user?

Using latest Tiptap with YJS.

**Comment by hanspagel** [comment, marked as answer]
That should work by default. Did you pass a hex color with 6 characters, e. g. #ffcc00? 

Or something like rgba or a shorter hex code? Those wouldn’t work.

**Reply by nc** [reply]
That works, thank you :)
